src/app/donation-start/donation-start-form/donation-start-form.component.html

```diff
@@ -196,8 +193,7 @@ <h3 class="b-rt-0 b-m-0 b-bold"><span class="span-hr"></span>How does Big Give u
             class="continue b-w-100 b-rt-0"
             mat-raised-button
             color="primary"
-            [disabled]="! donationForm.controls['amounts']!.valid"
-            (click)="next()"
+            (click)="progressFromStepOne()"
```

Comment:
Maybe worth feature-flagging all these changes, so we can merge this without having to do all 5 steps at once and get them perfect first time?

Would just need to set up the flag somehow either based on the environment or a URL param or something, then put back the old logic for disabling the button in case the flag is off, like so:

```typescript
            [disabled]="! donationForm.controls['amounts']!.valid && ! don819FlagEnabled"
            (click)="progressFromStepOne()"
```